tests/unit_tests/data_model/test_model_data_writer.py

```diff
@@ -1,45 +1,60 @@
 #!/usr/bin/python3
 
 import logging
-import re
 from pathlib import Path
 
+import astropy
```

Comment:
Same comment as above about importing all of astropy instead of just the exact exception and Table.

Reply:
See comment below.